crates/gitbutler-stack/src/stack.rs

```diff
@@ -542,6 +542,11 @@ impl Stack {
         }
 
         if self.heads.iter().all(|branch| branch.archived) {
+            // Remove previous PR information so it looks like we started a new stack
+            for head in self.heads.iter_mut() {
```

Comment:
Do you want to do this for all or only for the integrated ones?

Reply:
I think so. Getting rid of context too eagerly could be anoying